app/src/main/kotlin/com/wire/android/ui/home/conversations/common/ConversationItemTemplate.kt

```diff
@@ -18,9 +18,13 @@ fun ConversationItemTemplate(
     title: @Composable () -> Unit,
     subTitle: @Composable () -> Unit = {},
     eventType: EventType? = null,
-    onConversationItemClick: () -> Unit
+    onConversationItemClick: () -> Unit,
+    onConversationItemLongClick: () -> Unit
```

Comment:
Shouldn't we pass the `conversationId` so that we can identify later which conversation was clicked?

Reply:
I do it in another place, here I just pass the click and long item click event